builder/instance/builder.go

```diff
@@ -298,6 +299,7 @@ func (b *Builder) Run(ctx context.Context, ui packersdk.Ui, hook packersdk.Hook)
 			Ctx:                      b.config.ctx,
 			Debug:                    b.config.PackerDebug,
 			EbsOptimized:             b.config.EbsOptimized,
+			EnableNitroEnclave:       b.config.EnableNitroEnclave,
```

Comment:
Is there a specific reason why this was only added to the amazon-instance builder? 

It looks like this feature is only enabled for the instance builder. This same structure change to `awscommon.StepRunSourceInstance` needs to be added to the Ebs type builders in order to support Enclave for all builder types.

Reply:
That was just an oversight.  I had got it working with the instance builder and forgot to go back to the other ones.